### ISSUE TITLE
List without explicit height only shows first item.

### ISSUE THREAD

**Issue by izzmeifter** [BowlerHatLLC/feathers-sdk#36]
Lists that don't explicitly set either a height or layoutData the creates a height will only render the first item until the list is scrolled. If a list also has the verticalScrollPolicy off, the list will never render more than 1 item.

``` actionscript
<f:List id="detailsList" verticalScrollPolicy="off" horizontalScrollPolicy="off">
        <f:layoutData>
            <f:AnchorLayoutData left=0" right="0" top="0">
        </f:layoutData>
</f:List>
```

If you add either bottom="0" or a height property to the list, it will render fine, but in cases of dynamic data used to build a non scrollable list, this doesn't work.

**Comment by joshtynjala** [comment]
I can reproduce using the following code:

```
var group:LayoutGroup = new LayoutGroup();
group.autoSizeMode = AutoSizeMode.STAGE;
group.layout = new AnchorLayout();
this.addChild(group);

var list:List = new List();
list.dataProvider = new ListCollection(
[
]);
list.verticalScrollPolicy = ScrollPolicy.OFF;
list.horizontalScrollPolicy = ScrollPolicy.OFF;
list.layoutData = new AnchorLayoutData(0, 0, NaN, 0);
group.addChild(list);

var button:Button = new Button();
button.label = "Click Me";
button.addEventListener(Event.TRIGGERED, function():void
{
    list.dataProvider.addItem({label: "New Item"});
});
this.addChild(button);
```

This is probably the same issue as BowlerHatLLC/feathers#1394

**Comment by joshtynjala** [comment]
This is a regression, and it worked properly in 2.3.0, so I plan to include the fix in a 3.0.2 update. I will probably wait a week or two before releasing it, just in case other issues come up. Thanks for your patience.
